lib/beaker/host_prebuilt_steps.rb

```diff
@@ -13,7 +13,7 @@ module HostPrebuiltSteps
     SLEEPWAIT = 5
     TRIES = 5
     UNIX_PACKAGES = ['curl', 'ntpdate']
-    FREEBSD_PACKAGES = ['curl', 'perl5']
+    FREEBSD_PACKAGES = ['curl', 'perl5|perl']
```

Comment:
This will need attention if someone is going to implement e.g. docker hypervisor for freebsd, as these constants are used in ``beaker/hypervisor/docker.rb``. Currently they are used simply as:
```ruby
dockerfile += <<-EOF
...
apt-get install -y openssh-server openssh-client #{Beaker::HostPrebuiltSteps::CUMULUS_PACKAGES.join(' ')
..
EOF
```
etc.. Such a code is not able to handle our ``FREEBSD_PACKAGES`` which contain "alternatives" in form ``pkg1|pkg2``.